hoot-core/src/main/cpp/hoot/core/conflate/AbstractConflator.cpp

```diff
@@ -91,6 +93,25 @@ void AbstractConflator::_reset()
   _mergers.clear();
 }
 
+QString AbstractConflator::_matchSetToString(const MatchSet& matchSet) const
```

Comment:
A lot of refactoring of `AbstractConflator` to support doing match merging from `DiffConflator`.